lang/cs/Org.Apache.REEF.Common/Runtime/MachineStatus.cs

```diff
@@ -60,36 +61,56 @@ static MachineStatus()
                 InstanceName = processName
             };
         }
+#else
+        private const string NotSupportedValue = "-1";
```

Comment:
I think if we don't want to declare `NotSupportedValue` as declare it as `int` or `long` and keep it a string, its value can be more informative, e.g. `"<UNSUPPORTED>"` or something.